### PR TITLE
Adds definitions to backend kv template for health checking

### PR DESCRIPTION
**Pull request by zachomedia**
### Description

This PR adds definitions to the KV template to load backend health check information (path and interval).

I've only tested this change with the etcd backend.

### CODE REVIEW

File: templates/kv.tmpl [hunk 1]
```diff
@@ -19,6 +19,14 @@
     sticky = {{$sticky}}
 {{end}}
 
+{{$healthCheck := Get "" . "/healthcheck/" "path"}}
+{{$interval := Get "30s" . "/healthcheck/" "interval"}}
+{{with $healthCheck}}
```
Comment (emilevauge) [inline templates/kv.tmpl]: We may move `{{$interval := Get "30s" . "/healthcheck/" "interval"}}` below `{{with $healthCheck}}` ?

Comment (zachomedia) [inline templates/kv.tmpl]: I followed the same format as the load balancer line above (where `{{$sticky := Get "false" . "/loadbalancer/" "sticky"}}` is outside the `with`). I can move it inside the `{{with $healthCheck}}` if you want.

Comment (emilevauge) [inline templates/kv.tmpl]: It's not mandatory, but it would be better :)

Comment (zachomedia) [inline templates/kv.tmpl]: @emilevauge I moved the `interval` `Get` into the `with` block.

**Pull request merged**